nmdc_schema/migrators/migrator_from_X_to_PR19_and_PR70.py

```diff
@@ -0,0 +1,89 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+from nmdc_schema.migrators.helpers import load_yaml_asset
+import difflib
+import uuid
+
+
+class Migrator(MigratorBase):
+    r"""Migrates a database between two schemas."""
+
+    _from_version = "X"
+    _to_version = "PR19_and_PR70"
+
+    def upgrade(self):
+        r"""Migrates the database from conforming to the original schema, to conforming to the new schema."""
+
+        self.adapter.create_collection(collection_name="instrument_set")
+        self.fill_instrument_set()
```

Comment:
@turbomam These two lines build the `instrument_set` collection in mongo. You can view the `fill_instrument_set` function to see that is uses the yaml file to populate the `instrument_set`